src/ActionList/Item.tsx

```diff
@@ -205,27 +210,55 @@ const StyledItem = styled.div<
   &:hover ${StyledItemContent}::before,
   // - below Hovered
   // '*' instead of '&' because '&' maps to separate class names depending on 'variant'
-  :hover + * ${StyledItemContent}::before,
+  :hover + * ${StyledItemContent}::before {
+    // allow override in case another item in the list is active/focused
+    border-color: var(--item-hover-divider-border-color-override, transparent) !important;
+  }
+
   // - above Focused
   &:focus ${StyledItemContent}::before,
   // - below Focused
   // '*' instead of '&' because '&' maps to separate class names depending on 'variant'
   :focus + * ${StyledItemContent}::before,
   // - above Active Descendent
-  &.${itemActiveDescendantClass} ${StyledItemContent}::before,
+  &[${isActiveDescendantAttribute}] ${StyledItemContent}::before,
   // - below Active Descendent
-  .${itemActiveDescendantClass} + & ${StyledItemContent}::before {
+  [${isActiveDescendantAttribute}] + & ${StyledItemContent}::before {
     // '!important' because all the ':not's above give higher specificity
     border-color: transparent !important;
   }
 
-  // Focused OR Active Descendant
-  &:focus,
-  &.${itemActiveDescendantClass} {
+  // Active Descendant
+  &[${isActiveDescendantAttribute}='${activeDescendantActivatedDirectly}'] {
+    background: ${({focusBackground}) => focusBackground};
+  }
+  &[${isActiveDescendantAttribute}='${activeDescendantActivatedIndirectly}'] {
+    background: ${({hoverBackground}) => hoverBackground};
+  }
+
+  &:focus {
     background: ${({focusBackground}) => focusBackground};
     outline: none;
   }
 
+  &:active {
+    background: ${({focusBackground}) => focusBackground};
+    animation: actionList-item-active-bg 4s forwards ${get('animation.easeOutCubic')};
+
+    @keyframes actionList-item-active-bg {
+      0% {
+      }
+      50% {
+        transform: scale(1);
+        box-shadow: inset 0 0 0 rgba(${get('colors.scale.black')}, 0.04);
+      }
+      100% {
+        transform: scale(0.97);
+        box-shadow: inset 0 3px 9px rgba(${get('colors.scale.black')}, 0.04);
+      }
+    }
+  }
```

Comment:
@dgreif Unfortunately there are some rendering quirks when rendering a `transform` on delay like that. Unless there's a quick fix to avoid this 1px movement, we can just ignore it for now.

https://user-images.githubusercontent.com/293280/122989231-6bda0d80-d357-11eb-894a-84f0b5ec0899.mov

```suggestion
  &:active {
    background: ${({focusBackground}) => focusBackground};
  }
```

Reply:
I think using the same background as the active state will make this more consistent for now as well, so I'm good with reverting back to simply the background color